tools/rna_tools/vienna_rna/rnalfoldz.xml

```diff
@@ -0,0 +1,97 @@
+<tool id="rnalfoldz" name="RNALfoldz" version="2.1.6.0">
+    <description>calculates locally stable secondary structures of RNA</description>
+    <expand macro="requirements" />
+    <expand macro="version_command" />
+    <expand macro="stdio" />
+    <macros>
+        <token name="@EXECUTABLE@">RNALfold</token>
+        <import>macros.xml</import>
+    </macros>
+    <command>
+<![CDATA[
+        RNALfold -L $span -T $temperature -d$dangling -z$zthresh < $fasta_input | sed -r 's/(^[.()]*) +\((.*)\) +(.*) +z= +(.*)/\1\t\2\t\3\t\4/' > $out_file
```

Comment:
The only three arguments of sed that are POSIX standard are `-e`, `-f`, `-n`

http://pubs.opengroup.org/onlinepubs/9699919799/utilities/sed.html

If there is a way to achieve the same without `-r` it would be even better